FModel/Creator/Bases/UCreator.cs

```diff
@@ -43,13 +43,13 @@ protected UCreator(UObject uObject, EIconStyle style)
     protected readonly SKPaint DisplayNamePaint = new()
     {
         IsAntialias = true, FilterQuality = SKFilterQuality.High,
-        Typeface = Utils.Typefaces.DisplayName, TextSize = _NAME_TEXT_SIZE,
+        Typeface = Utils.Typefaces.DisplayName ?? Utils.Typefaces.Default, TextSize = _NAME_TEXT_SIZE,
```

Comment:
You already set all of those to Default in Typefaces.cs

Reply:
Ah yeah. Before I added the specific typefaces for Multiversus, it crashed because of the DisplayName and Description typefaces being null so I just put that. Might be good to keep that check for games that don't have their own typefaces implemented yet

Reply:
I think it can be done directly in Typeface.cs
Actually that file needs a rework, especially to use OnTheFly more often